### PR TITLE
issue#5

### PR DESCRIPTION
**Pull request by robertb98**
'Upon login, user's email should be displayed in the header.' will be added after combining front+back

### CODE REVIEW

File: backend/Dockerfile [hunk 1]
```diff
@@ -11,5 +11,5 @@ RUN pip install pipenv && pipenv install
 
 COPY . /backend
 
-EXPOSE 8000
+# EXPOSE 8000
```
Comment (robertmusil) [inline backend/Dockerfile:14]: don't leave commented out code

File: frontend/src/views/Login.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+const Login = () => {
+  const classes = useStyles();
+  const { register, errors: errorsForm, handleSubmit } = useForm({
+    reValidateMode: "onSubmit",
+  });
+
+  const errorsBackend = useSelector((state) => state.alert.errors);
+
+  const onSubmit = (data) => {
+    console.log(data);
```
Comment (robertmusil) [inline frontend/src/views/Login.js:38]: So this doesn't log you in, it's just a template now, so it doesn't complete the ticket, does it?

Comment (robertb98) [reply frontend/src/views/Login.js:38]: Yes, for now it's only a template, how the /login will look like. The functionality will be in the next branch combining issue#5+issue#6.

Comment (nitz14) [inline frontend/src/views/Login.js:38]: do we need this console log in code ?

File: frontend/src/views/Login.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+  };
+
+  const errors = combineErrors(errorsForm, errorsBackend);
+
+  const validation = {
+    email: {
+      required: "Required field",
+    },
+    password: {
+      required: "Required field",
```
Comment (nitz14) [inline frontend/src/views/Login.js:48]: Can we create some variable with this text ?

File: frontend/src/views/Register.js [hunk 4]
```diff
@@ -49,26 +58,26 @@ const Register = () => {
...
       validate: (value) => {
-        return value === watch("password2") || "Passwords don't match";
+        return value === watch("password2") || PASSWORDS_DONT_MATCH;
       },
     },
     password2: {
       minLength: {
         value: 8,
-        message: "Required more than 8 chars",
+        message: NEED_MORE_THAN_8_CHARS,
```
Comment (robertmusil) [inline frontend/src/views/Register.js:76]: could be slightly nicer with `message: MIN_LENGHT(8)'

File: frontend/src/views/Login.js [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+
+const Login = () => {
+  const classes = useStyles();
+  const { register, errors: errorsForm, handleSubmit } = useForm({
+    reValidateMode: "onSubmit",
+  });
+
+  const errorsBackend = useSelector((state) => state.alert.errors);
+
+  const onSubmit = (data) => {};
```
Comment (robertmusil) [inline frontend/src/views/Login.js:38]: so this doesn't close the issue before this actually connects to backend, no?

Comment (robertb98) [reply frontend/src/views/Login.js:38]: The connection frontend=backend will be implemented in the next PR.

**Pull request closed without merging**